src/controllers/UserController.ts

```diff
@@ -50,9 +50,11 @@ export class UserController {
       }
     }
 
-    @Put("/:id")
-    put(@Param("id") id: string): string {
-       return "Updating the user " + id;
+    @Put("")
+    @Authorized()
+    put(@CurrentUser() user: User, @Body() body: User): string {
+      Container.get(UserService).update(body.email, body.fullName, body.notifications, user);
```

Comment:
If you dont change the update function you can just do update(user.username, body)